pkg/resources/grant_privileges_to_account_role_acceptance_test.go

```diff
@@ -0,0 +1,909 @@
+package resources_test
+
+import (
```

Comment:
did you check the old resource with the new tests (ofc beside the new behavior)?